Makefile

```diff
@@ -68,7 +68,8 @@ ALL = \
 	$(ISO_ROOT)/images/$(KUBE_CONTROLLER_MANAGER_IMAGE_NAME)-$(KUBE_CONTROLLER_MANAGER_IMAGE_VERSION).tar.gz \
 	$(ISO_ROOT)/images/$(KUBE_PROXY_IMAGE_NAME)-$(KUBE_PROXY_IMAGE_VERSION).tar.gz \
 	$(ISO_ROOT)/images/$(KUBE_SCHEDULER_IMAGE_NAME)-$(KUBE_SCHEDULER_IMAGE_VERSION).tar.gz \
-	$(ISO_ROOT)/images/$(NGINX_IMAGE_NAME)-$(NGINX_IMAGE_VERSION).tar.gz
+	$(ISO_ROOT)/images/$(NGINX_IMAGE_NAME)-$(NGINX_IMAGE_VERSION).tar.gz \
+	$(ISO_ROOT)/images/$(SALT_MASTER_IMAGE_NAME)-$(SALT_MASTER_IMAGE_VERSION).tar.gz
```

Comment:
In future, please always keep a trailing `\` on the last line, and an empty line below (as there is one right now).

Rationale: when adding a new value to this list, one wouldn't need to edit two lines (like above), but only add one, which is easier to validate/review.